DataProviderService/StarSystemSqLiteRepository.cs

```diff
@@ -164,16 +163,13 @@ public List<StarSystem> GetOrCreateStarSystems(string[] names, bool fetchIfMissi
 
         public StarSystem GetOrFetchStarSystem(string name, bool fetchIfMissing = true, bool refreshIfOutdated = true)
         {
-            if (name == string.Empty) { return null; }
-            return GetOrFetchStarSystems(new[] { name }, fetchIfMissing, refreshIfOutdated)?.FirstOrDefault();
+            return name == string.Empty ? null : GetOrFetchStarSystems(new[] { name }, fetchIfMissing, refreshIfOutdated)?.FirstOrDefault();
```

Comment:
The code is equivalent, but as a matter of style I find it more maintainable to have the "guard" statements in a separate block at the beginning, and the meat of the function below. It makes it easier to distuingish the two, think about them separately, and refine either block separately. So I'd keep this change as it was.